fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingMutuallyByRecords.java

```diff
@@ -371,7 +373,9 @@ private CompletableFuture<Boolean> buildNextRangeOnly(List<Range> missingRanges,
             }
             if (rangeToBuild != null) {
                 infiniteLoopProtection(rangeToBuild, missingRanges);
-                return iterateAllRanges(null,
+                final List<Object> additionalLogMessageKeyValues = new ArrayList<>(Arrays.asList(LogMessageKeys.CALLING_METHOD, "mutualMultiTargetIndex"));
+                additionalLogMessageKeyValues.addAll(fragmentLogMessageKeyValues());
```

Comment:
Should this add all of `indexingLogMessageKeyValues`?

Reply:
It's fine to defer until later, but it does seem like `indexingLogMessageKeyValues` should be included automatically by `iterateAllRanges`, and most places that logs happen (which should include fragment information)